data/person/01K5ST4BCF6P4NQ9W4DW7S2V5H.toml

```diff
@@ -10,7 +10,3 @@ type = "congress"
 congress = 19
 position = "representative"
 
-[[memberships]]
```

Comment:
Should still be part of the 20th congress:

https://en.wikipedia.org/wiki/20th_Congress_of_the_Philippines